src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -12,28 +13,27 @@ public boolean hasSomebodyWon() {
         return (hasPlayerWonSet(playerOne, playerTwo) || hasPlayerWonSet(playerTwo, playerOne));
     }
 
-    public boolean hasSomeoneOfPlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
-        return (playerWhoIsWinning.getSetScore() - playerWhoIsLoosing.getSetScore() == 2);
+    public boolean hasOneOfThePlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLosing) {
+        return (playerWhoIsWinning.getSetScore() - playerWhoIsLosing.getSetScore() == 2);
     }
 
     public boolean hasPlayerWonSet(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
         if (playerWhoIsWinning.getSetScore() == 11 && playerWhoIsLoosing.getSetScore() < 10) {
             return true;
 
         } else if (playerWhoIsWinning.getSetScore() >= 11 && playerWhoIsLoosing.getSetScore() >= 10) {
-            return hasSomeoneOfPlayersTwoPointsMore(playerWhoIsWinning, playerWhoIsLoosing);
+            return hasOneOfThePlayersTwoPointsMore(playerWhoIsWinning, playerWhoIsLoosing);
```

Comment:
Isn't it "_The first player to reach 11 points wins that game, however a game must be won by at least a two point margin._"?

If so, wouldn't this method be simpler like that:
````java
public boolean hasPlayerWonSet(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
        return playerWhoIsWinning.getSetScore() >= 11 && hasOneOfThePlayersTwoPointsMore(playerWhoIsWinning, playerWhoIsLoosing);
    }
````